lib/Agrammon/DB/Datasets.pm6

```diff
@@ -5,33 +5,36 @@ use Agrammon::DB::User;
 use Agrammon::Timestamp;
 
 class Agrammon::DB::Datasets does Agrammon::DB {
-    has Str $.version;
+    has %.agrammon-variant;
```

Comment:
I do wonder if this and the `!variants` private method want lifting out to some role.

Reply:
Done